pkg/crypto/certloader/fswatcher/fswatcher_test.go

```diff
@@ -34,9 +33,7 @@ func Test(t *testing.T) {
 }
 
 func (s *FsWatcherTestSuite) TestWatcher(c *C) {
-	tmp, err := ioutil.TempDir("", "")
-	c.Assert(err, IsNil)
-	defer os.RemoveAll(tmp)
+	tmp := c.MkDir()
```

Comment:
`MkDir()` says:
>  Create a new temporary directory which is automatically removed after  the suite finishes running.

is the directory removed recursively as `os.RemoveAll()` does above? had a look at the code: https://github.com/go-check/check/blob/10cb98267c6cb43ea9cd6793f29ff4089c306974/check.go#L170 but could not figure out if the dir is recursively removed or not.

[... checks a bit more ...]

OK, seems that it is should work as desired:
https://github.com/go-check/check/blob/10cb98267c6cb43ea9cd6793f29ff4089c306974/check.go#L637